src/app/Instance/CreateInstance/CreateInstance.tsx

```diff
@@ -31,6 +31,8 @@ export interface CreateInstanceProps {
     onSuccess: () => void,
     onError: (error: CreateInstanceError) => void
   ) => void;
+  /** The parent container to append the dialog to. Defaults to "document.body". */
+  appendTo?: () => HTMLElement;
```

Comment:
I had to add the `appedTo` prop to make sure the dialog is rendered inside storybook preview `div` and then use [play functions](https://storybook.js.org/docs/react/writing-stories/play-function) to simulate user interaction.